src/Entity/Speaker.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\SpeakerRepository")
+ */
+class Speaker
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @Assert\Length(max=255)
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @Assert\Length(max=5)
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=5)
+     */
+    private $title;
+
+    /**
+     * @Assert\Email()
+     * @Assert\Length(max=255)
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=255)
+     */
+    private $email;
+
+    /**
+     * @Assert\NotBlank()
+     * @ORM\Column(type="text")
+     */
+    private $biography;
+
+    /**
+     * @Assert\Length(max=255)
+     * @Assert\NotBlank()
+     * @ORM\Column(type="string", length=255)
+     */
+    private $photo;
+
+    /**
+     * @Assert\Url()
+     * @Assert\Length(max=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $twitter;
+
+    /**
+     * @Assert\Url()
+     * @Assert\Length(max=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $facebook;
+
+    /**
+     * @Assert\Url()
+     * @Assert\Length(max=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $linkedin;
+
+    /**
+     * @Assert\Url()
+     * @Assert\Length(max=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $github;
+
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    public function getName(): ?string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name): self
```

Comment:
Remove setters, use a constructor instead

Reply:
I need them for DTO (see src/Controller/Speaker/Edit.php::handle)

Reply:
No you do not :) You could change the code there and make a builder or something in your repository class or in a dedicated class. Don't bother with that now though